build/__tools__/run-tests.sh

```diff
@@ -239,14 +246,6 @@ if [ $? == 0 ]; then
     die mock contract test succeeded though it should have failed
 fi
 
-#-----------------------------------------
-yell run tests with the eservice database
-#------------------------------------------
-
-say run various pdo scripts - test-request, test-contract, create, update, shell - using database
```

Comment:
you removed the tests which use a name rather than a URL and --service-name seems gone.   Seems to me names instead of URLs seemed a nice feature (easier to remember, allows semantic description, urls can change ...). And if we don't use them, why do we still keep them in the DB?

PS: Looking now further down through the code i guess code still has the ability to query enclaves by name, just no higher-level code makes use of it and there are no tests anymore?

Reply:
Looks like providing the names is no longer an option during an update operation (in test-request, test-contract and shell send). It would be good to be able to specify update enclaves by name at least via shell if not via the test scripts. Otherwise, currently names seem unusable by the application during any update operation (more than just the fact that there are no tests making using of names)

Reply:
correct. for tests, only URLs.